src/amber/cli/templates/app/config/initializers/database.cr.ecr

```diff
@@ -3,7 +3,7 @@ require "granite_orm/adapter/<%= @database %>"
 
 Granite::ORM.settings.database_url = Amber.settings.database_url
 Granite::ORM.settings.logger = Amber.settings.logger
```

Comment:
@drujensen Currently with this code overwrites the `Server` logger:

```
03:47:00 Granite    | (INFO) [Amber 0.6.3] serving application "Forum" at http://0.0.0.0:8080
03:47:00 Granite    | (INFO) Server started in production.
03:47:00 Granite    | (INFO) Startup Time 00:00:00.000150000
03:47:03 Granite    | (INFO) SELECT topics.id, topics.title, topics.content, topics.published FROM topics : []
03:47:04 Granite    | (INFO) SELECT users.id, users.name, users.password FROM users : []
03:47:05 Granite    | (INFO) SELECT pets.id, pets.name, pets.owner, pets.type FROM pets : []
^C03:47:08 Granite    | (INFO) Shutting down Amber
```

Please change it to:

```
Granite::ORM.settings.logger = Amber.settings.logger.dup
```